index.html

```diff
@@ -56,6 +56,119 @@ <h1 id="main-heading">
+          <h3 class="work-title">Multi-Post Stories Gain+Glory</h3>
+          <ul class="language-tags">
+            <li class="tag">Ruby on rails</li>
+            <li class="tag">JavaScript</li>
+            <li class="tag">CSS</li>
+            <li class="tag">HTML</li>
+          </ul>
+          <a href="#" class="work-link">
+            See Project
+          </a>
+        </article>
+        <article class="work">
+          <img src="./media/multipost.png" alt="first project screenshot" class="work-image">
+          <h3 class="work-title">Multi-Post Stories Gain+Glory</h3>
+          <ul class="language-tags">
+            <li class="tag">Ruby on rails</li>
+            <li class="tag">JavaScript</li>
+            <li class="tag">CSS</li>
+            <li class="tag">HTML</li>
+          </ul>
+          <a href="#" class="work-link">
+            See Project
+          </a>
+        </article>
+        <article class="work">
+          <img src="./media/printing.png" alt="first project screenshot" class="work-image">
+          <h3 class="work-title">Multi-Post Stories Gain+Glory</h3>
+          <ul class="language-tags">
+            <li class="tag">Ruby on rails</li>
+            <li class="tag">JavaScript</li>
+            <li class="tag">CSS</li>
+            <li class="tag">HTML</li>
+          </ul>
+          <a href="#" class="work-link">
+            See Project
+          </a>
+        </article>
+      </section>
+      <div id="about-heading-wrap">
+        <h1 id="about-heading">
+          About Me
+        </h1>
+      </div>
+      <section id="about">
+        <p id="about-info">
+          I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+          don’t hestiate to contact me.
+        </p>
+        <a href="#" id="resume-link">
+          Get My Resume
+        </a>
+      </section>
+      <section id="tech-stack">
+        <article class="stack">
+          <img src="./media/icon-languages.png" alt="" class="tech-image">
+          <h3 class="tech-heading">Languages</h3>
+          <ul class="tech-list">
+            <li class="tech">Ruby on rails</li>
+            <li class="tech">JavaScript</li>
+            <li class="tech">CSS</li>
+            <li class="tech">HTML</li>
+          </ul>
+        </article>
+        <article class="stack">
+          <img src="./media/icon-frameworks.png" alt="" class="tech-image">
+          <h3 class="tech-heading">Frameworks</h3>
+          <ul class="tech-list">
+            <li class="tech">Bootstrap</li>
+            <li class="tech">Ruby on rails</li>
+            <li class="tech">RSpec</li>
+            <li class="tech">Capybara</li>
+            <li class="tech">Selenium</li>
+          </ul>
+        </article>
+        <article class="stack">
+          <img src="./media/icon-skills.png" alt="" class="tech-image">
```

Comment:
[OPTIONAL] I suggest you fix your margins and paddings for this section. I may have approved your pull request but the previous reviewer forgot to highlight that your margins are too big. It however is a [core requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) of this project
<img width="1660" alt="Screenshot 2022-10-29 at 10 36 44" src="https://user-images.githubusercontent.com/48546992/198824768-efd27164-3db5-45b2-a48f-822741d87f6a.png">
This is your project:
<img width="683" alt="Screenshot 2022-10-29 at 12 38 41" src="https://user-images.githubusercontent.com/48546992/198824830-c2adb6d3-4996-47b0-b547-dea30ccb813c.png">
<img width="687" alt="Screenshot 2022-10-29 at 12 38 53" src="https://user-images.githubusercontent.com/48546992/198824832-35fc5dea-32d6-41d8-bf0d-d383b80b8592.png">

This is the figma design. 
<img width="377" alt="Screenshot 2022-10-29 at 12 44 31" src="https://user-images.githubusercontent.com/48546992/198824881-47a2982f-461c-461d-8ad2-ed8099cf6a3e.png">

Please fix this to match figma so that in the next milestone your project is already fixed and wont be sent back for a review.